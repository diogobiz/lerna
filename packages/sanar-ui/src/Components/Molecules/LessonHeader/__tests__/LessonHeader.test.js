
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESLessonHeader from '../'

it('renders correctly', () => {
    const component = (
        <ESLessonHeader>
            <h1>ESLessonHeader</h1>
        </ESLessonHeader>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
