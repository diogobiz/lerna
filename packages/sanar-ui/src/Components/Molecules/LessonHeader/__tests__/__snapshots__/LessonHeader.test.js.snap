// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-row-flex ant-row-flex-space-between ant-row-flex-bottom es-row es-lesson-header"
  id="es-lesson-header"
  style={Object {}}
>
  <div
    className="ant-col es-col es-lesson-header__bookmark no-rate"
    style={Object {}}
  />
  <div
    className="ant-col es-col es-lesson-header__center ant-col-xs-24 ant-col-md-24 ant-col-lg-8 ant-col-lg-push-1"
    style={Object {}}
  >
    <h1>
      ESLessonHeader
    </h1>
  </div>
</div>
`;
